client/src/partials/reports/modals/balance.modal.html

```diff
@@ -41,6 +41,19 @@
     </div>
   </div>
 
+  <!-- accounts option choosen -->
```

Comment:
Just so you know, bootstrap has classes for radio buttons.

[Documentation](http://getbootstrap.com/css/#checkboxes-and-radios)